aspnet/tutorials/nano-server.rst

```diff
@@ -1,8 +1,149 @@
+
+In this tutorial, you'll take an existing ASP.NET 5 application and deploy it to a Nano Server instance running IIS.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+Introduction
+------------
+
+Nano Server is a new version of Windows Server with a small footprint optimized for the cloud. You can learn more about Nano Server from the official docs `available here <https://msdn.microsoft.com/en-us/library/mt126167.aspx>`_.
+
+In this tutorial, we will using a pre-built Virtual Hard Disk (VHD) for Nano Server which can be downloaded `here <https://msdn.microsoft.com/en-us/virtualization/windowscontainers/nano_eula>`_. This pre-built VHD already includes the Reverse Forwarders and IIS packages which are required for this tutorial.
+
+Before proceeding with this tutorial, you will require the published output of an existing ASP.NET 5 application. Refer to the guidance on :ref:`dnu-publish`. Ensure your application is built targeting the **64-bit** version of ``coreclr``.
+
+Setting up the Nano Server Instance
+-----------------------------------
+
+Create a new Virtual Machine using Hyper-V on your development machine using the previously downloaded VHD. The machine will require you to set an administator password before logging on. At the VM console, press F11 to set the password before the first logon.
+
+After setting the local password, you will manage Nano Server using the management tools remotely.
+
+Connecting to your Nano Server Instance using PowerShell Remote
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Open an elevated PowerShell window to add your remote Nano Server instance to your ``TrustedHosts`` list.
+
+.. code:: ps1
+
+  $ip = "10.83.181.14" # replace with the correct IP address
+  Set-Item WSMan:\localhost\Client\TrustedHosts "$ip" -Force
+
+Once you have added your Nano Server instance to your ``TrustedHosts``, you can connect to it using PowerShell Remote session
+
+.. code:: ps1
+
+  $ip = "10.83.181.14" # replace with the correct IP address
+  $s = New-PSSession -ComputerName $ip -Credential ~\Administrator
+  Enter-PSSession $s
+
+If you have successfully connected using PSRemote, you prompt will look like this ``[10.83.181.14]: PS C:\Users\Administrator\Documents>``
+
+Installing the HTTPPlatformHandler Module
+-----------------------------------------
+
+At the moment, the process to install the HTTPPlatformHandler Module for IIS is manual. You will need to install the latest 64-bit version of the `HttpPlatformHandler <http://www.iis.net/downloads/microsoft/httpplatformhandler>`_ on a regular (not Nano) machine. After installing you will need to copy the following files:
```

Comment:
Add a sentence or two stating what the HttpPlatformHandler is and why it is needed.